src/pyodide/python-entrypoint-helper.ts

```diff
@@ -142,9 +147,17 @@ function makeHandler(pyHandlerName: string): Handler {
         'prep_python',
         async () => await preparePython()
       );
-      return await enterJaegerSpan('python_code', () => {
+
+      const result = await enterJaegerSpan('python_code', () => {
         return mainModule[pyHandlerName].callRelaxed(...args);
       });
+
+      // Support returning a pyodide.ffi.FetchResponse.
+      if (typeof result.js_response !== 'undefined') {
```

Comment:
```suggestion
      if (result.js_response !== undefined) {
```

Reply:
Probably we want to use `?.` so that if the handler incorrectly returns `None` we don't raise an error here.